tests/test_symbolic_coefficients.py

```diff
@@ -119,6 +119,83 @@ def test_coefficients_w_xreplace(self):
 
         assert expected == str(eq.evaluate.lhs)
 
+    # FIXME: Needs to test several grid spacings and dimensions
+    @pytest.mark.parametrize('order', [1, 2, 6, 8])
+    @pytest.mark.parametrize('extent', [1., 10., 100.])
+    @pytest.mark.parametrize('conf', [{'l': 'NODE', 'r1': 'x', 'r2': None},
+                                      {'l': 'NODE', 'r1': 'y', 'r2': None},
+                                      {'l': 'NODE', 'r1': '(x, y)', 'r2': None},
+                                      {'l': 'x', 'r1': 'NODE', 'r2': None},
+                                      {'l': 'y', 'r1': 'NODE', 'r2': None},
+                                      {'l': '(x, y)', 'r1': 'NODE', 'r2': None},
+                                      {'l': 'NODE', 'r1': 'x', 'r2': 'y'}])
+    def test_default_rules_equation(self, order, extent, conf):
```

Comment:
I would have a pure simple symbolic test first, i.e just check lhs/rhs of the symbolic evaluated equation.

Reply:
As in do `str(expr)` of each `Eq` and check for equality?

Reply:
Yes, for example
https://github.com/devitocodes/devito/blob/5938e7f198f8eb9cfd5e8889c8ee9ba203eb4a44/tests/test_derivatives.py#L342

Reply:
Is there some way to get the coefficients as float? The only roadblock is that the symbolic coefficients default to fractions, whilst the standard coefficients are in floating point form, so comparison obvs returns False

Reply:
Done in the latest push